### PR TITLE
Add ETH Sepolia

### PR DESCRIPTION
**Pull request by noahlitvin**
* Waiting on Pool/Router addresses from Chainlink before merging
* This should be run without `upgrade-from` (Should be a fresh deployment, do we need to change salt?)

### CODE REVIEW

File: omnibus-sepolia.toml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+[setting.ccip_token_pool]
+defaultValue = "0x9b65749b38278060c5787cce0391ac7f1094c8e8"
+
+# TODO clean this block after deployment
+[invoke.unapprove_wrongfully_deployed_snx_pool]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+func = "removeApprovedPool"
+args = ["2"]
```
Comment (noisekit) [inline omnibus-sepolia.toml:64]: this is very unnecessary.
```suggestion
```

File: omnibus-sepolia.toml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    "tomls/oracles/eth.toml",
+    "tomls/collaterals/weth.toml",
+    "tomls/pools/spartan-council.toml",
+    "tomls/permit-all-createPool.toml",
+    "tomls/permit-all-registerMarket.toml",
+    "tomls/permit-all-transferCrossChain.toml",
+]
+
+[setting.snx_package]
+defaultValue = "synthetix:3.3.5"
```
Comment (noisekit) [inline omnibus-sepolia.toml:18]: shouldn't we update to the latest?

Comment (dbeal-eth) [reply omnibus-sepolia.toml:18]: no it should match mainnet I think to start?

Comment (noisekit) [reply omnibus-sepolia.toml:18]: Yeah. This is just a revert of deleted deployment. Did not realise that at the time

File: omnibus-sepolia.toml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+name = "synthetix-omnibus"
```
Comment (noisekit) [inline omnibus-sepolia.toml:1]: Actually... lets enable test too before merge please. This should be an upgrade

**Pull request closed without merging**